static/lang/re-en.json

```diff
@@ -3355,6 +3355,99 @@
+                },
+                "TracingSigil": {
+                    "Description": "When you cast @UUID[Compendium.pf2e.spells-srd.Item.AUctDF2fqPZN2w4W]{Sigil}, you can modify the spell to add the following to its standard effects: The target leaves a magical trail that you and others can try to follow. You and other creatures can attempt to @UUID[Compendium.pf2e.actionspf2e.Item.EA5vuSgJfiHH7plD]{Track} the target, substituting an Arcana, Nature, Occultism, or Religion check (whichever matches the magical tradition of your sigil spell) for the Survival check. Much like Tracking with Survival, this must take place somewhere the target has been and follow the trail; it doesn't allow anyone to find the target from a distance. You can only have a single target marked with a modified sigil in this way. If you use this ability again on a second target, the sigil spell on the first target ends, and your mark fades."
+                },
+                "UneasyRest": {
+                    "Description": "When you cast @UUID[Compendium.pf2e.spells-srd.Item.o4lRVTwSxnOOn5vl]{Sleep}, you can modify the spell to add the following to its standard effects: Subjects of your spell experience troubled dreams. When a target that failed or critically failed its saving throw wakes up, it is @UUID[Compendium.pf2e.conditionitems.Item.TBSHQspnbcqxsmjL]{Frightened 1}."
+                },
+                "VolatileGrease": {
+                    "Description": "When you cast @UUID[Compendium.pf2e.spells-srd.Item.Wu0xFpewMKRK3HG8]{Grease}, you can modify its target to be 1 creature. If you do, replace the spell's standard effects with the following: You splash the target with combustible grease. The target must attempt a @Check[type:reflex] save.",
+                    "CriticalSuccess": "The target is unaffected.",
+                    "Failure": "As success, except the weakness to fire lasts for 1 minute.",
+                    "Success": "The target is splattered with grease and gains weakness 2 to fire until the end of your next turn. The target or an adjacent creature can rub off the combustible grease with an Interact action, ending the effect.",
+                    "TitleHeightened": "Heightened (+2)",
```

Comment:
Let's make a new generic list of these. We have the dynamic ones in en.json but those rely on more structured data. So a generic set ala what the amp heightened has would be best.